pkg/porter/cnab.go

```diff
@@ -68,6 +59,19 @@ func (o *bundleFileOptions) Validate(cxt *portercontext.Context) error {
 		o.Dir = cxt.FileSystem.Abs(o.Dir)
 	}
 
+	if o.File != "" {
+		if o.Dir != "" && !filepath.IsAbs(o.File) {
+			o.File = cxt.FileSystem.Abs(filepath.Join(o.Dir, o.File))
```

Comment:
Isn't o.Dir always set in the if statement above? I think we can simplify this to the following
```suggestion
		if !filepath.IsAbs(o.File) {
			o.File = cxt.FileSystem.Abs(filepath.Join(o.Dir, o.File))
```

Reply:
I thought the `o.Dir` is only resolved if it's not empty in the if statement above?

Reply:
Ah okay thanks I thought that if it was empty, we were setting it to the absolute path of the current working directory. If we did that then we wouldn't need a bunch of defensive checks later in the code since we know it was initialized properly. What do you think?